eva-server/src/test/java/uk/ac/ebi/eva/server/ws/GeneWSServerTest.java

```diff
@@ -66,7 +66,8 @@ public void setUp() throws Exception {
         String alternate = "T";
         String vepVersion = "88";
         String vepCacheVersion = "89";
-        testVariantEntity = new VariantWithSamplesAndAnnotation(chromosome, start, end, reference, alternate);
+        String mainId = "rs1";
+        testVariantEntity = new VariantWithSamplesAndAnnotation(chromosome, start, end, reference, alternate, mainId);
```

Comment:
Check that the main Id has been returned, because the equals method does not compare it.